packages/trpc/server/routers/viewer/organizations/updateUser.handler.ts

```diff
@@ -59,35 +60,9 @@ export const updateUserHandler = async ({ ctx, input }: UpdateUserOptions) => {
   }
 
   // Is requested user a member of the organization?
-  const requestedMember = await prisma.membership.findFirst({
-    where: {
-      userId: input.userId,
-      teamId: organizationId,
-      accepted: true,
-    },
-    include: {
-      team: {
-        include: {
-          children: {
-            where: {
-              members: {
-                some: {
-                  userId: input.userId,
-                },
-              },
-            },
-            include: {
-              members: true,
-            },
-          },
-        },
-      },
-      user: {
-        select: {
-          username: true,
-        },
-      },
-    },
+  const requestedMember = await MembershipRepository.getRequestedMember({
+    userId: input.userId,
```

Comment:
Ensure consistency in naming conventions for variables and parameters [medium]

- Generated by [LGTM](https://lgtm.pro)